libs/providers/flagd/src/lib/flagd-provider.spec.ts

```diff
@@ -1,23 +1,27 @@
 import { FlagdProvider } from './flagd-provider';
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
-import { OpenFeature, Client, ErrorCode, StandardResolutionReasons } from '@openfeature/nodejs-sdk';
-
+import {
+  OpenFeature,
+  Client,
+  ErrorCode,
+  StandardResolutionReasons,
+} from '@openfeature/nodejs-sdk';
 
 describe('FlagdProvider', () => {
-  const host = "http://localhost"
-  const port = 8080
+  const host = 'http://localhost';
+  const port = 8080;
   it('should be and instance of FlagdProvider', () => {
     expect(new FlagdProvider()).toBeInstanceOf(FlagdProvider);
   });
 
   describe('http service tests', () => {
     const axiosMock = new MockAdapter(axios);
-    let client: Client
+    let client: Client;
 
     beforeEach(() => {
       OpenFeature.setProvider(new FlagdProvider());
-      client = OpenFeature.getClient("test")
+      client = OpenFeature.getClient('test');
```

Comment:
Do we need linting in this repo?

Reply:
https://github.com/open-feature/node-sdk-contrib/issues/88